styles.css

```diff
@@ -0,0 +1,674 @@
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background: #ec5242;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(5px);
+    opacity: 0.92;
+    transition: 1s;
+  }
+
+  .barlist li {
+    margin: 14px 10px;
+    list-style: none;
+  }
+
+  .Worksection {
+    background: #e5e5e5;
+  }
+
+  .menu-text {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  #close {
+    background-color: transparent;
+    border: 0;
+    color: whitesmoke;
+    cursor: pointer;
+    font-size: 40px;
+    position: absolute;
+    right: 40px;
+    z-index: 9;
+  }
+}
+
+/* Desktop */
+
+@media screen and (min-width: 768px) {
+  .flist {
+    background-color: black;
+    display: flex;
+    justify-content: flex-end;
+    margin: 0;
+    gap: 20px;
+  }
+
+  .event {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 121px;
+    padding-left: 130px;
+  }
+
+  .fa-facebook-f {
+    color: #d3d3d3;
+  }
+
+  .flist a {
+    text-decoration: none;
+    color: #d3d3d3;
+    display: flex;
+    list-style: none;
+  }
+
+  .logo a {
+    display: flex;
+    color: black;
+    text-decoration: none;
+    place-items: center;
+    padding-top: -8px;
+    padding-left: 50px;
+  }
+
+  .bar {
+    display: none;
+  }
+
+  .lnav {
+    background-color: white;
+    height: 50px;
+    padding-top: 5px;
+    padding-left: 10px;
+    padding-right: 10px;
+    justify-content: space-between;
+  }
+
+  .barlist {
+    display: flex;
+    gap: 35px;
+    justify-content: flex-end;
+    list-style: none;
+    margin-top: -54px;
+    line-height: 20px;
+  }
+
+  .barlist li {
+    list-style: none;
+    margin: 14px 10px;
+  }
+
+  .barlist a {
+    text-decoration: none;
+    color: black;
+    display: flex;
+    list-style: none;
+  }
+
+  .barlist a.active {
+    color: var(--secondary-dark);
+  }
+
+  .barlist a:hover {
+    color: #ec5242;
+  }
+
+  .barlist button {
+    border: #ec5242 2px solid;
+    background-color: white;
+    color: #ec5242;
+    font-size: 18px;
+  }
+
+  .barlist button:hover {
+    background-color: #ec5242;
+    color: white;
+  }
+
+  .hello {
+    padding: 10px 40px;
+    display: flex;
+  }
+
+  .hello h1 {
+    font-size: 30px;
+    font-weight: 700;
+    color: #ec5242;
+  }
+
+  .hello h2 {
+    text-transform: uppercase;
+    font-weight: 900;
+    line-height: 55px;
+    font-family: sans-serif;
+    background-image: linear-gradient(rgba(255, 38, 0, 0.76), rgba(255, 51, 0, 0.575)), url("images/lbackground.jpg");
+    color: transparent;
+    -webkit-background-clip: text;
+    font-size: 45px;
+  }
+
+  .hello p {
+    background-color: #f5f5f5;
+    border: #d3d3d3 1px solid;
+    padding: 10px;
+    font-size: 17px;
+    line-height: 30px;
+    width: 550px;
+    font-weight: 500;
+    color: #272a31;
+    font-family: "Lato", sans-serif;
+    margin: 20px;
+  }
```

Comment:
for the headline section in the desktop view please fix the following:

- [ ] Kindly add the correct margin-top and margin-left values for the whole headline section to match the given design.
- [ ] Please decrease the width of the main heading to match the given design.
- [ ] Please decrease the margin between the two `<h3>` heading to match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170763039-ac0f129f-458b-48f7-926e-fb4ad9380fb1.png)  | ![image](https://user-images.githubusercontent.com/68796232/170763000-35919967-367c-4d36-8271-c48dccd819cb.png)  |